src/prefect/storage/bitbucket.py

```diff
@@ -80,14 +94,85 @@ def get_flow(self, flow_name: str) -> "Flow":
         Returns:
             - Flow: the requested flow
         """
+
+        return (
+            self._get_flow_cloud(flow_name)
```

Comment:
I have a minor preference for `_get_flow_from_bitbucket_cloud` / `_get_flow_from_bitbucket_server` to make it clear we are not referring to Prefect Server/Cloud

Reply:
Done